app/code/core/Mage/Checkout/Block/Cart/Sidebar.php

```diff
@@ -273,12 +268,9 @@ public function getCacheTags()
 
     /**
      * Processing block html after rendering
-     *
-     * @param   string $html
-     * @return  string
      */
     #[\Override]
-    protected function _afterToHtml($html)
+    protected function _afterToHtml(string $html): string
```

Comment:
Two questions:

1) Should this be in this PR?

2) Are we deciding to start adding PHP types on methods? It does break any block that overrides this method.

Reply:
you're right, I think I ended up on that code for some reason related to this PR, but then probably something got solved some other way and I left this part thinking it was harmless but... better to revert all of them